iceberg/iceberg-handler/src/test/queries/negative/alter_acid_table_to_iceberg_failure.q

```diff
@@ -3,4 +3,4 @@ set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
 
 drop table tbl_orc;
 create table tbl_orc (a int, b string) stored as orc tblproperties('transactional'='true');
-alter table tbl_orc set tblproperties ('storage_handler'='org.apache.iceberg.mr.hive.HiveIcebergStorageHandler');
\ No newline at end of file
+alter table tbl_orc convert to iceberg;
```

Comment:
Could we please cover all of the aspects mentioned below:

````
To migrate a Hdfs Hive format table to an Iceberg table use the command:

ALTER TABLE [dbname.]table_name CONVERT TO ICEBERG [TBLPROPERTIES(...)];

For example
     - ALTER TABLE hive_table CONVERT TO ICEBERG;
     - ALTER TABLE hive_table CONVERT TO ICEBERG TBLPROPERTIES(
       'iceberg.catalog' = 'hadoop.catalog');
     - ALTER TABLE hive_table CONVERT TO ICEBERG TBLPROPERTIES(
       'metadata.generator.threads' = '10');

The input table must meet these requirements:
     - table is an external table
     - table is not a transactional table
     - InputFormat must be either PARQUET, ORC, or AVRO

The data files of the table are not affected by this operation.

When you migrate a Hive table <table_name> to Iceberg, a backup of the table, named <table_name>backup, is created.
````

Reply:
The TBLPROPERTIES stuff is there in the ``TestHiveIcebergMigration``, and all other validation cases as well. I changed to use our covert stuff. Creating new table as we discussed offline doesn't look feasible and is more or less like a copy